lib/bugsnag/integrations/rails/rails_breadcrumbs.rb

```diff
@@ -0,0 +1,134 @@
+require "bugsnag/breadcrumbs/breadcrumbs"
+
+module Bugsnag::RailsBreadcrumbs
```

Comment:
This is fine, but I've had to move the `require` statement for the file due to namespace clashes.  (As having a `Bugsnag::Rails` namespace makes it search for `Rails::Railtie` at `Bugsnag::Rails::Railtie`)

Reply:
In that case, it would be preferable to use `::Rails::Railtie` instead of `Rails::Railtie` in lib/bugsnag/integrations/railtie.rb to avoid possible issues.